ow_power_system/config/system.cfg

```diff
@@ -1,51 +1,83 @@
 # configuration
-efficiency: 0.9
+efficiency: 0.9			# %
 
-# initials
-initial_power: 0.0          # watts
-initial_voltage: 4.1        # volts
-initial_temperature: 20.0   # deg. C
+
+# Initial battery configuration
+
+initial_power: 0.0			# watts
+initial_voltage: 25.2			# volts
+initial_temperature: 20.0   		# deg. C
+
+initial_soc: 0.95           		# % (used for first placeholder publication)
 
 # voltage
-base_voltage: 2.8           # volts
-voltage_range: 0.1          # volts
+base_voltage: 16.8			# volts
+voltage_range: 0.1			# volts
 
 # power
-baseline_power: 1.0         # watts
+baseline_power: 6.0			# watts
 
 # temperature
-min_temperature: 17.5       # deg. C
-max_temperature: 21.5       # deg. C
-
-# battery life
-battery_lifetime: 2738.0    # Estimate of battery lifetime (seconds)
-
-# GSAP (prognostics engine) related configurations
+min_temperature: 17.5			# deg. C
+max_temperature: 21.5			# deg. C
 
-# GSAP processes data at a certain rate.
-gsap_rate: 0.5              # hz
+# Estimate of battery lifetime
+battery_lifetime: 27380.0		# s
 
-# Power fault profiles (found in ../profiles) are CSV files whose rows
-# may represent any time interval. It is typical for the interval to
-# be 1Hz. To synchronize the consumption of these profiles with
-# GSAP's processing time of 0.5 Hz, we use every Nth profile line,
-# where N = profile interval/gsap_rate. For now, we avoid additional
-# user specification and computation by providing the required value
-# of N here.
-profile_increment: 2
 
-# Similarly, since the inputs used to compute power consumption come
-# from the /joint_states topic published at 50Hz, we divide this by
-# GSAP's rate to determine the number of entries used in a moving
-# average:
-power_average_size: 100
+# Power system related configurations
 
-# The current GSAP model breaks if power input is to high.  As a
-# workaround to prevent this, we cap its power input.
-max_gsap_power_input: 20    # watts
+# The frequency of the main loop in the power system.
+# Dictates the time interval between publications
+# & the size of the mechanical power moving average window.
+loop_rate: 1				# hz
 
 # Number of threads to use for the asynchronous ROS spinning in the main
 # power system loop (to allow the joint states callback function to run
-# at the same 50Hz publication rate, rather than the loop's 0.5Hz rate).
+# at the same 50Hz publication rate, rather than the loop's rate).
 spinner_threads: 4
 
+
+# GSAP (prognostics engine) related configurations
+
+# Maximum RUL estimation for prognoser predictions.
+max_horizon: 100000			# s
+
+# Number of samples created by each node during prognoser predictions for accuracy.
```

Comment:
It definitely affects the return time of prognoser calls, though I will say that I'm not really sure it actually influences performance. If the predictions complete sooner due to a lower horizon, it doesn't really matter because another one will start as soon as the next batch of data is received. The only way it will noticeably and consistently affect performance is if we reduce this value to something infeasibly low such that the predictions take almost no time at all.

I know I said in the last PR that it affected performance but that was before my discussions with Chris & Chetan. I'll update the comment to note that the value affects the rate of prediction returns, but I don't think a note on performance is as necessary.

Reply:
I agree. In general I think it's good to document the default return time for a prognoser, and how this configuration affects that. If I understand correctly, this is essentially the response time of the battery prognoser, correct?

Reply:
The max horizon? No, that's the cap on how high the RUL prediction can go. RUL is heavily tied to prognoser response time though.

A 'default' return time isn't really something that's documented because it depends on the current RUL predictions (e.g. a prediction with an RUL of 3000 will return much faster than a prediction with an RUL of 80000). The max horizon just caps the possible RUL values and thus is tied to a maximum possible return time, but I doubt it is fully consistent.

Essentially, the return time of GSAP's prognosers will change as the simulation goes on; generally they will get faster since RUL will steadily decrease towards 0 over time.

Reply:
That makes sense! The additional comment looks good and is appreciated.